CONTRIBUTING.md

```diff
@@ -27,24 +23,6 @@ All new features should enter `main` through a pull request.
   the merge (if we were really committed to the gag this would use the word _may,_
   but it's not that funny).
 
-## Non-production deployments
```

Comment:
I'd say we should document what we actually are doing, but there was never much evidence that people read this, so...

Reply:
we do have some documentation on the subject over at https://github.com/usds/prime-simplereport-docs

We also do not have evidence that any one reads the existing docs